### PR TITLE
{vis}[intel-2016a] OpenCV 3.1.0 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (3 easyconfigs in this PR)
node2006.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e30bbff12fb0ebaa530a9cc1c52f2dc7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2602.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/885dd5b91907508483ddce0743318c64 for a full test report.

File: easybuild/easyconfigs/o/OpenCV/OpenCV-3.1.0-intel-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+source_urls = ['https://github.com/Itseez/opencv/archive/']
+sources = ['%(version)s.zip']
+
+patches = ['OpenCV-%(version)s_with_IPP.patch']
+
+builddependencies = [
+    ('CMake', '3.5.2'),
+]
+
+javaver = '1.8.0_92'
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenCV/OpenCV-3.1.0-intel-2016a.eb]: @hajgato please use `%(javaver)s` everywhere?

**Comment by hajgato** [comment]
@boegel %(javaver)s and SHLIB_EXT used

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ec80a987c84a34ad273d13b95323fde0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2408.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/02e1973b14f1c49b413c8fd2d78b1008 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**